gooddata-pandas/gooddata_pandas/result_convertor.py

```diff
@@ -5,7 +5,7 @@
 from attrs import define, field, frozen
 
 from gooddata_sdk import BareExecutionResponse, ExecutionResult
-from gooddata_sdk.compute.model.execution import ResultSize
+from gooddata_sdk.compute.model.execution import ResultCacheMetadataWrapper, ResultSizeBytes, ResultSizeDimensions
```

Comment:
If I am not mistaken you added these classes in gooddata-sdk/gooddata_sdk/init.py meaning you can use from gooddata_sdk to import them.

Reply:
fixed